### PR TITLE
Use the TRAVIS_BUILD_NUMBER as the version number

### PR DESCRIPTION
**Pull request by yamlin**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,5 @@
 version = 0.0.6-unofficial
+versionPrefix = 0.0.0.
```
Comment (chinkong83) [inline gradle.properties]: I think two zero is enough, do we need three?

**Comment by chinkong83** [comment]
LGTM

File: gradle.properties [hunk 2]
```diff
@@ -1,4 +1,5 @@
 version = 0.0.6-unofficial
```
Comment (shichung) [inline gradle.properties]: if "version = 0.0.6-unofficial" is no longer in use, it could be cleaned up

**Pull request merged**